evalml/tests/automl_tests/test_automl_search_classification.py

```diff
@@ -1267,16 +1256,19 @@ def test_automl_passes_allow_long_running_models(
         X_train=X,
         y_train=y,
         problem_type="multiclass",
+        objective="Log Loss Multiclass",
         allow_long_running_models=allow_long_running_models,
         automl_algorithm=algo,
-        max_batches=2,
+        max_batches=3,
```

Comment:
What was the reason for this change?

Reply:
`DefaultAlgorithm` doesn't call `filter_estimators()` and drops the long running estimators until the final non-ensembling batch (batch 2 if starting from batch 0) so this previous test case was not actually covering the `DefaultAlgorithm` case. IIRC it was obfuscated by the patches in the previous implementation of this test. With `AutoMLTestEnv` adding a batch shouldn't cause much overhead if we add a batch as well!